samples/cpp/embedded-speech/samples/settings.cpp

```diff
@@ -238,7 +238,7 @@ bool VerifySettings()
                     }
                     else
                     {
-                        return model->Name.compare(modelName) == 0;
+                        return model->Name.compare(modelName) == 0 || model->Locales[0].compare(modelName) == 0;
```

Comment:
>Locales [](http://example.com/codeflow?start=77&length=7)

If this is like windows sr and tts locales, it is possible to specify "en-US", "en-GB", or just "en" which would match any regional sub codes like "-US" or "-GB".

If that true here, and do you need to handle this?

Reply:
Speech recognition locales are full-length, like "en-US". See https://learn.microsoft.com/azure/cognitive-services/speech-service/language-support?tabs=stt

The short form like "en" is for translation target languages:
https://learn.microsoft.com/azure/cognitive-services/speech-service/language-support?tabs=speech-translation